web/pages/charity/index.tsx

```diff
@@ -88,10 +89,12 @@ export default function Charity(props: {
     () =>
       charities.filter(
         (charity) =>
-          charity.name.toLowerCase().includes(query.toLowerCase()) ||
-          charity.preview.toLowerCase().includes(query.toLowerCase()) ||
-          charity.description.toLowerCase().includes(query.toLowerCase()) ||
-          (charity.tags as string[])?.includes(query.toLowerCase())
+          searchInAny(
+            query,
+            charity.name,
+            charity.preview,
+            charity.description
+          ) || (charity.tags as string[])?.includes(query.toLowerCase())
```

Comment:
I guess charity tags are already lowercase?

Reply:
actually charities just don't have tags (except "Featured"). I never got around to it :/